switchres_wrapper.cpp

```diff
@@ -34,6 +35,11 @@ MODULE_API void sr_init() {
 	swr->parse_config("switchres.ini");
 }
 
+MODULE_API void sr_load_ini(char config[20]) {
+	swr->parse_config(config+'.'+'i'+'n'+'i');
+
+}
```

Comment:
Can you:
  * change the parameter to `const char*`
  * remove the forced `.ini` extension
  * remove that empty line